tests/sentry/api/test_utils.py

```diff
@@ -51,3 +51,13 @@ def test_no_params(self):
         start, end = get_date_range_from_params({}, optional=True)
         assert start is None
         assert end is None
+
+    @freeze_time('2018-12-11 03:21:34')
+    def test_relative_date_range(self):
```

Comment:
Could be worth testing having `statsPeriod(Start|End)` missing

Reply:
done